src/App.js

```diff
@@ -1,11 +1,37 @@
+/* eslint-disable no-shadow */
 import React from 'react';
-import style from './App.module.scss';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import {
+  increment, decrement, addToDo, removeToDo, resetTodo,
+} from './store/counterData/actions';
+import Counter from './Counter';
+import ToDo from './ToDo';
 
-const App = () => {
-  const mth = 'Hello  my World';
-  return (
-    <h3 className={style.smth}>{mth}</h3>
-  );
-};
+const App = ({
+  // eslint-disable-next-line react/prop-types
+  counter, toDoList, increment, decrement, addToDo, removeToDo, resetTodo,
+}) => (
+  <div>
+    <Counter counter={counter} increment={increment} decrement={decrement} />
+    <ToDo toDoList={toDoList} addToDo={addToDo} removeToDo={removeToDo} resetTodo={resetTodo} />
```

Comment:
please change names of properties to more informative names. (names of components, properties)
what is **removeToDo**? it will be better to give another name (removeListItem etc.) clearToDoList...